README.md

```diff
@@ -122,10 +122,12 @@ Alpaca-py has a lot of client classes. There is a client for each API and even a
 
 **Market Data API:**  ``StockHistoricalDataClient``, ``CryptoHistoricalDataClient``, ``CryptoDataStream``, ``StockDataStream``
 
+**News API:** ``NewsClient``, ``NewsDataStream``
```

Comment:
Can we please treat that news data is part of Market Data API? Therefore, appending `NewsClient` and `NewsDataStream` into Market Data API would be nice.